### DISCUSSION TITLE
CSS IntelliSense not behaving as expected

### DISCUSSION THREAD

**Discussion by kmarler18** [tailwindlabs/tailwindcss#10262] [Help]
Hey Everybody!

I installed Tailwind CSS IntelliSense and it's not behaving like I expected. For example, no hover preview, autocomplete, etc. 

I went through the [troubleshooting steps](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss#troubleshooting) as best I could (though it's possible I may have overlooked something as I'm new to Tailwind and coding in general).

I also found [this article](https://javascript.plainenglish.io/how-to-fix-tailwind-css-intellisense-in-visual-studio-code-3dede794df21) and tried implementing his techniques into my settings.json file to no avail.

![Screen Shot 2023-01-06 at 8 03 48 PM](https://user-images.githubusercontent.com/96091818/211124009-2b48a129-e0e1-4a75-9227-b4efa0a6834a.png)

Any thoughts?

As always, thank you.
Kyle

**Comment by jayantghadge** [comment]
If you have the PostCSS extension enabled, disable it and give it a try.

**Reply by kmarler18** [reply]
Hi @jayantghadge 

To the best of my knowledge, I do not have a PostCSS extension downloaded so there's nothing to disable there.

It might be worth mentioning my process. Perhaps I'm misunderstanding how to get plugins into a Tailwind project. 

I'm simply going to the Extension marketplace in Visual Studio Code and downloading it, and I thought that was all I had to do. 

![Screen Shot 2023-01-09 at 12 52 15 AM](https://user-images.githubusercontent.com/96091818/211248301-e9db63bf-0728-442f-ab53-778ac6ce3e19.png)


I'm using the "[Play CDN](https://tailwindcss.com/docs/installation/play-cdn)" as well. (Not sure if that has any bearing.) Do I need to add some code to the tailwind.config section? I haven't added anything there for plugins. Maybe I need to?

![Screen Shot 2023-01-09 at 1 04 30 AM](https://user-images.githubusercontent.com/96091818/211249088-1a7d901d-817c-4eec-acf8-be46871c22e7.png)



Thank you for your input.

**Reply by jayantghadge** [reply]
Generally, downloading extension from marketplace is enough to make it work.
As you mentioned that you used CDN, make sure that there is a tailwind.config.js. If it's not there then create one in the root directory.
